include/core/bdf.h

```diff
@@ -117,5 +117,60 @@ number_of_previous_solutions(
     return 0;
 }
 
+/**
+ * @brief Extrapolates vector of solution to time_vector[0] using previous solution times and previous solution
+ *
+ * @tparam The type of the variable being extrapolated (e.g. double, Tensor<1,dim>)
+ * @param time_vector The vector of times. The solution will be extrapolated to time 0. It should be of size number_of_previous_solutions+1
+ * @param solution_vector The vector of solutions.The solution will be extrapolated to index 0. It should be at least of size number_of_previous_solutions
+ * @param number_of_previous_solution The number of previous solutions to use to extrapolate
```

Comment:
```suggestion
 * @param number_of_previous_solutions The number of previous solutions to use to extrapolate
```

and the following calls of this variable